src/ResourceManager/HDInsight/Commands.HDInsight/ManagementCommands/AddAzureHDInsightComponentVersionCommand.cs

```diff
@@ -47,12 +47,12 @@ public class AddAzureHDInsightComponentVersionCommand : HDInsightCmdletBase
 
         public override void ExecuteCmdlet()
         {
-            ConfirmAction("Adding Component Version", Name,
+            ConfirmAction("Adding Component Version", "AzureHDInsightConfig",
```

Comment:
No, the resource being modified is the AzureHDInsightConfig because there's no ComponentName object